inference-engine/tests/functional/plugin/cpu/shared_tests_instances/subgraph_tests/mvn_multiply_add.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "subgraph_tests/mvn_multiply_add.hpp"
+
+using namespace SubgraphTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecision = {
+        InferenceEngine::Precision::FP32
+};
+
+std::vector<InferenceEngine::Precision> idxPrecision = {
+        InferenceEngine::Precision::I64
+};
+
+const std::vector<bool> acrossChannels = {
+        true,
+        false
+};
+
+const std::vector<bool> normalizeVariance = {
+        true,
+        false
+};
+
+const std::vector<float> epsilon = {
+        0.000000001
+};
+
+const std::vector<std::string> epsMode = {
+        "inside_sqrt",
+        "outside_sqrt"
+};
+
+const std::vector<std::tuple<InferenceEngine::SizeVector, InferenceEngine::SizeVector>> shapes_1D = {
+        { std::vector<size_t>{5}, std::vector<size_t>{5}},
+        { std::vector<size_t>{64}, std::vector<size_t>{64}},
+};
+
+const std::vector<std::tuple<InferenceEngine::SizeVector, InferenceEngine::SizeVector>> shapes_2D = {
+        { std::vector<size_t>{1, 5}, std::vector<size_t>{1, 5}},
+        { std::vector<size_t>{2, 17}, std::vector<size_t>{1, 17}},
+        { std::vector<size_t>{9, 64}, std::vector<size_t>{1, 64}},
+        { std::vector<size_t>{5, 15}, std::vector<size_t>{2, 15}},
```

Comment:
This two shapes can not be broadcastable. You can see [here](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/broadcast_rules.md) for the rules. Thanks.

Reply:
Yes, you're right. Thank you!